catalog/imaging/imaging-app/src/main/resources/features.xml

```diff
@@ -28,7 +28,7 @@
         <bundle dependency="true">mvn:org.codice.alliance.catalog.core/catalog-core-api/${project.version}</bundle>
         <bundle>mvn:org.codice.alliance.imaging/imaging-nitf-api/${project.version}</bundle>
         <bundle>mvn:org.codice.alliance.imaging/imaging-nitf-impl/${project.version}</bundle>
-        <bundle>mvn:org.codice.alliance.imaging/imaging-service-api/${project.version}</bundle>
+        <bundle dependency="true">mvn:org.codice.alliance.imaging/imaging-service-api/${project.version}</bundle>
```

Comment:
I don't recall why originally made this change, but I reverted it and tested the code. Everything seems fine without it.